### ISSUE TITLE
Issues with kubernetes health_check

### ISSUE THREAD

**Issue by DevOps-Hipcam** [opf/openproject-helm-chart#3]
Hello, how are you? I need your help to resolve an issue with openproject and kubernetes.

if you set OPENPROJECT_HTTPS=true, it is not possible to pass the health checks, so the pods keep in failing state.

          livenessProbe:
            httpGet:
              path: /health_check
              port: 8080
            initialDelaySeconds: 120
          readinessProbe:
            httpGet:
              path: /health_check
              port: 8080
            initialDelaySeconds: 120

the kubernetes healt_checks works by http, and when you enable **OPENPROJECT_HTTPS=true**, the response of the pods is **301** redirect to https and the probe keep failing. The result is that the pod doesn't pass the checks and keep failing.

```
curl -v http://10.3.33.124:8080/health_check
*   Trying 10.3.33.124:8080...
* Connected to 10.3.33.124 (10.3.33.124) port 8080 (#0)
> GET /health_check HTTP/1.1
> Host: 10.3.33.124:8080
> User-Agent: curl/7.78.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
**< HTTP/1.1 301 Moved Permanently**
< Content-Type: text/html
< Location: https://10.3.33.124:8080/health_check
< X-Frame-Options: SAMEORIGINn"

```

We needed to change the probes to tcpsocket only (that is working), but it is recommended to keep in httpget.

if you set **OPENPROJECT_HTTPS=false**, the livenessProbe is working fine (the endpoint health_check doesn't redirect and return 200) but you keep with this screen.

![image](https://user-images.githubusercontent.com/118916840/206280460-8838db09-5861-4695-b65c-e782a3a482f7.png)

It is importante that the tls termination is in internal AWS Load balancer.

**To sum up:** We need this option enables **OPENPROJECT_HTTPS=true** recommended from openproject community, and keep the livenessprobe with **httpget** working.

Could you check that please?

@machisuji 

Thanks in advance

**Comment by machisuji** [comment]
Hello @DevOps-Hipcam, thanks for the report. That's a good point. We're looking into it!

**Comment by DevOps-Hipcam** [comment]
Thanks a lot!! @machisuji. Let me know any update!!

**Comment by machisuji** [comment]
Hello @DevOps-Hipcam - We've moved our helm chart over to https://github.com/opf/helm-charts.
The chart has been rewritten. In the new version this issue is fixed.

It's best to do a fresh install of the new charts if you want to update your current setup.
Sorry for the hassle.

In the meantime you could also fix the probes yourself in the existing, checked out files (this repo).
You simply have to change the probe path from `/health_check` to `/health_checks/default`.
This will work both with http and https.

I'm closing this issue.
